languagetool-language-modules/ga/pom.xml

```diff
@@ -0,0 +1,95 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.languagetool</groupId>
+    <artifactId>languagetool-parent</artifactId>
+    <version>4.9-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>language-ga</artifactId>
+  <url>http://www.languagetool.org</url>
+  <name>Irish module for LanguageTool</name>
+
+  <licenses>
+    <license>
+      <name>GNU Lesser General Public License</name>
+      <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt</url>
+      <distribution>repo</distribution>
+      <comments>The license refers to the source code, resources may be under different licenses</comments>
+    </license>
+  </licenses>
+
+  <packaging>jar</packaging>
+
+  <developers>
+    <developer>
+      <name>Jim O'Regan</name>
+      <roles><role>Maintainer</role></roles>
+    </developer>
+    <developer>
+      <name>Daniel Naber</name>
+      <roles><role>Maintainer</role></roles>
+    </developer>
+    <developer>
+      <name>Marcin Miłkowski</name>
+      <roles><role>Maintainer</role></roles>
+    </developer>
+  </developers>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+          <excludes>
+            <exclude>org/languagetool/resource/ga/earraidi-ga.bs</exclude>
+            <exclude>org/languagetool/resource/ga/eile-ga.bs</exclude>
+            <exclude>org/languagetool/resource/ga/filter-gramadoir.pl</exclude>
+          </excludes>
+      </resource>
+    </resources>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.languagetool</groupId>
+      <artifactId>languagetool-core</artifactId>
+      <version>${languagetool.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.languagetool</groupId>
+      <artifactId>hunspell-native-libs</artifactId>
```

Comment:
Is this still needed? We got rid of this dependency for all other languages recently (in 4.8).

Reply:
No, it's not. Removed.